src/test/java/chess/domain/board/TestOtherTeamBoard.java

```diff
@@ -0,0 +1,15 @@
+package chess.domain.board;
+
+import static chess.domain.TestPieces.*;
+
+import java.util.Map;
+
+import chess.domain.piece.Piece;
+import chess.domain.position.Square;
+
+public class TestOtherTeamBoard implements BoardGenerator {
+    @Override
+    public Map<Square, Piece> generate() {
+        return Map.of(new Square("c3"), WHITE_QUEEN, new Square("d4"), BLACK_QUEEN);
+    }
+}
```

Comment:
각각의 경우에 맞는 generator가 만들어 지고 있는데요.
아래와 같이 하면 하나만 만들어서 다양한 곳에서 사용할 수 있지 않을까요?

```suggestion
public class TestOtherTeamBoard implements BoardGenerator {
    private final Map<Squard, Piece> board;
    
    public TestBoardGenerator(Map<Square, Piece> board) {
        this.board = board;
    }

    @Override
    public Map<Square, Piece> generate() {
        return board;
    }
}
```

Reply:
반영했습니다 :) 웹에서도 save 된 board 를 불러오는데 사용할 수 있어서 좋더라구요 ㅎㅎ